cookbooks/travis_postgresql/recipes/ci_server.rb

```diff
@@ -35,6 +35,18 @@
   mode 0o755
 end
 
+file '/lib/systemd/system/postgresql.service' do
+  action :delete
+  notifies :run, 'execute[daemon-reload]', :immediately
+  only_if { node['lsb']['codename'] == 'xenial' }
+end
+
+execute 'daemon-reload' do
```

Comment:
Because this resource is specific to systemctl, it should either be renamed or use the `name` attribute, like so:

``` ruby
execute 'systemctl daemon-reload' do
  action :nothing
  only_if { node['lsb']['codename'] == 'xenial' }
end
```